### PR TITLE
govc: add sso.group commands

### LINKED ISSUE #1483: Add support to create/list sso groups and add sso users to the existing groups
To use govc to manage sso user/groups, we want something like this:
sso.group.ls
sso.group.create
sso.group.rm
sso.group.update  //which will add user to the existing groups
sso.group.permission.set //this may be already available by sso.permission.set, just double check

cc @dougm and thanks a lot!

### PR DESCRIPTION
**Pull request by dougm**
Fixes #1483

### CODE REVIEW

**Comment by dougm** [comment]
@apalcloud @AndrewDi can you review?

**Comment by apalcloud** [comment]
> @apalcloud @AndrewDi can you review?

Thanks @dougm. A quick question, does govc support add/remove AD/Windows domain groups/users?

File: govc/sso/group/rm.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+func (cmd *rm) Usage() string {
+	return "NAME"
+}
+
+func (cmd *rm) Description() string {
+	return `Remove SSO group.
+
+Examples:
+  govc sso.user.rm NAME`
```
Comment (AndrewDi) [inline govc/sso/group/rm.go]: rename `govc sso.user.rm NAME` to `govc sso.group.rm NAME`

Comment (dougm) [reply govc/sso/group/rm.go]: Good catch, fixed.

**Comment by AndrewDi** [comment]
LGTM

**Comment by dougm** [comment]
@apalcloud no, the API govc uses doesn't support any admin operations for external directories.

**Comment by apalcloud** [comment]
> @apalcloud no, the API govc uses doesn't support any admin operations for external directories.
Ok thx

**Pull request merged**